File: Source/NeoAiSystems.com/NeoDesktop/Entity/StoryIdObject.php
<?php
  namespace TApp;
  class TStoryIdObject extends \sky\TEntity {
    protected function DefineProperties(){
      parent::DefineProperties();
      $this->AddProperties(array(
        "SQLiteId" => array("check" => "Integer"),
        "MySQLId" => array("check" => "Integer")
      ));
    }
  }
?>

File: Source/NasX/entity/storyidobject.pas
unit StoryIdObject;

{$mode objfpc}{$H+}

interface

uses
  Entity;

type
  { TStoryIdObject }
  TStoryIdObject = class(TEntity)
  private
    FMySQLId: Integer;
    FSQLiteId: Integer;
  published
    property SQLiteId: Integer read FSQLiteId write FSQLiteId;
    property MySQLId: Integer read FMySQLId write FMySQLId;
  end;

implementation

{ TStoryIdObject }

initialization
  TStoryIdObject.RegisterEntityClass;

end.
